src/gc/gchandletable.cpp

```diff
@@ -22,3 +22,8 @@ void GCHandleTable::Shutdown()
 {
     Ref_Shutdown();
 }
+
+Object* GCHandleTable::ObjectFromHandle(OBJECTHANDLE handle)
+{
+    return OBJECTREFToObject(::HndFetchHandle(handle));
```

Comment:
When you look at what HndFetchHandle actually does, it gets AppDomain index, and then validates a bunch of things on the AppDomain. All these are bad dependencies for local GC. Have you thought about what we want to about those?

I am thinking that the whole validation should stay on the VM side, and the only method on the GC/EE interface should be a method to fetch AppDomain index from handle table. (It would be nice to not call it AppDomain index, but something more general - like `void* context`.)